### DISCUSSION TITLE
Make a nav: subcategory always collapsed

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6442] [Q&A] [answered]
Hello, is it possible to indicate for a section of mkdocs.yml:nav to always be collapsed while keeping others opened?

e.g this Linux category:

https://github.com/squidfunk/mkdocs-material/assets/82747632/6227465c-bc81-4bb0-9003-76ee068627fb

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello,
I don't think so. Once you enable the `navigation.expand` feature every navigation section will get expanded.
However, you could disable that feature, and later add your own code to manage navigation expansion.
https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript

We do so on the Gothic Modding Community Page:
https://github.com/Gothic-Modding-Community/gmc/blob/dev/overrides/assets/javascripts/extra.js#L151-L196
However, I'm not sure if the code is the best, as it changes the `checked` state of the input elements, and perhaps this should be limited to only the active navigation element 🤔
https://github.com/squidfunk/mkdocs-material/blob/d363f903b863df0990b49627804debc4e730e589/src/templates/assets/javascripts/patches/indeterminate/index.ts#L60-L85
The `navigation.expand` feature changes the `indeterminate` attribute as fas as I understand.
